CHANGELOG.md

```diff
@@ -2,6 +2,9 @@
 
 ## VNext
 
+
+
+## Version 2.18.0-beta1
```

Comment:
might be good to start a new practice of adding date as well to changelog.
eg:
https://github.com/open-telemetry/opentelemetry-dotnet/blob/main/src/OpenTelemetry/CHANGELOG.md#110-beta2